### ISSUE TITLE
Setting a property to null in the initial state

### ISSUE THREAD

**Issue by mergebandit** [apollographql/apollo-link-state#115]
Pretty simple. Lets say I'm setting up some initial state for a list of pages to create links from a dashboard. Each page contains a name (for the link label), as well as a brand and *optional* pageName. I'm struggling to figure out how to handle nullable values. Here's a sample

```
export default {
  Query: {
    pages: () => [
      {
        name: 'My Awesome Home Page',
        brand: 'awesomeBrand',
        pageName: null
      }
    ]
  }
};
```

If I query with this:

```
const DASHBOARD_QUERY = gql`
  query GetPages {
    pages @client {
      name
      brand
      pageName
    }
  }
`;
```

the value for `pageName` comes back as `"null"` instead of `null`. Is there a better way to do this than check in the `props`  method of the `graphql` HoC whether my `pageName === "null"`

**Comment by jbaxleyiii** [comment]
oh whoa! That should be happening? Would you be able to write a failing test for this in a PR?

**Comment by mergebandit** [comment]
Sure tried to. Turns out this was related to `next` and `next-routes`. Discovered this after your tests passed with flying colors. 

On a related note, one of the things I ran into with your tests ... if I return an object from one of the resolvers, the response contains a `Symbol(id)` field in the returned object. Obviously would be difficult to write tests for my app if I had to account for that ... any suggestions?
